python/rascaline/tests/utils/correlate_density.py

```diff
@@ -261,6 +261,7 @@ def test_lambda_soap_vs_powerspectrum():
     lsoap = calculator.compute(density)
     keys = lsoap.keys.remove(name="o3_lambda")
     keys = keys.remove("o3_sigma")
+    keys = keys.remove("order_nu")
```

Comment:
should we rename this to `body_order` instead?

Reply:
We could do, but would require a change everywhere. Different PR?

Reply:
And perhaps something to be discussed with others too, "correlation_order" / "order_nu" feels more CG-like, but "body_order" more intuitive / better UX. Something that can be easily changed later but I think not necessary now. Thoughts?

Reply:
I don't know, this feels trivial enough to change now, except if we don't agree on the new naming?

Reply:
Since this was internal before, the name did not matter too much. But if we are exposing it, we should give it a nice name!

Reply:
Fair enough, I'll change it